packages.txt

```diff
@@ -164,6 +166,7 @@ libusb1
 libuuid
 libxslt
 linuxHeaders
+llvm
```

Comment:
`llvm` is apparently LLVM 4 which doesn't compile for me.

Reply:
Okay, lets strip this for now.